spec/models/in_person_enrollment_spec.rb

```diff
@@ -491,4 +494,36 @@
       expect(failed_enrollment_without_notification.eligible_for_notification?).to eq(false)
     end
   end
+
+  describe 'enhanced_ipp?' do
+    before do
+      allow(IdentityConfig.store).to receive(:usps_eipp_sponsor_id).and_return(usps_eipp_sponsor_id)
+    end
+
+    context 'when the enrollment sponsor ID is equal to the EIPP sponsor ID' do
+      it 'returns true' do
+        user = create(:user)
```

Comment:
I'm not used to seeing FactoryBot `create` statements inside of it blocks.  Would you be up for moving this up to the context block?

Reply:
`create` is no longer inside the `it` block. Resolved with commit 01eff13e11a461d1485a125175e0e38dbdac6cf1